docs/community/documentation-style-guide.rst

```diff
@@ -0,0 +1,377 @@
+chronological order for each version of the module.
+This allows users to easily and quickly see what changes have been made for
+each release.
+
+Some useful tips for the changelog:
+
+* make it human readable
+* stick to one subsection for each version
+* use the following tags to describe changes:
+
+  * Adds, for any new features
+  * Changes, for any changes in the current functionality or code base
+  * Deprecates, for anything that will be removed in future releases
+  * Removes, for anything that was removed in the current release
+  * Fixes, for any bug fixes
+  * Security, for any security-related changes
+
+Who should I address?
+^^^^^^^^^^^^^^^^^^^^^
+This file addresses all types of users.
+
+
+CONTRIBUTING.rst
+================
+What is it?
+^^^^^^^^^^^
+The contribution guidelines defined in this file serve to communicate how people
+should contribute to the project.
+It helps them open useful issues and make well-formed Pull Requests that
+conform to the expectations of the project maintainers.
+It lists the types of contributions one may make for the particular project
+and how to do it.
+It also includes a quick Get Started section that explains the basic
+steps going from forking to submitting a Pull Request.
+Finally, it lists the Pull Request guidelines (see `Invenio PR guidelines <https://invenio.readthedocs.io/en/latest/community/contribution-guide.html>`_).
+
+Who should I address?
+^^^^^^^^^^^^^^^^^^^^^
+This file addresses new contributors and developers.
+
+INSTALL.rst
+===========
+What is it?
+^^^^^^^^^^^
+The INSTALL file provides the user with installation instructions.
+It should contain a brief description of the installation process
+and one or more commands to install the module.
+It also specifies any available install options.
+
+Who should I address?
+^^^^^^^^^^^^^^^^^^^^^
+This file addresses all types of users.
+
+
+README.rst
+==========
+What is it?
+^^^^^^^^^^^^^^^^^^^^^
+The README file should contain a very broad, big picture view of what
+the module provides along with a bulleted list of its features.
+It's recommended to avoid lengthy abstracts, try to keep it to 3 or 5 sentences.
+Engage the reader by describing as soon as possible:
+
+- what the module does
+- how it can be used
+- and why it should be used
+
+Who should I address?
+^^^^^^^^^^^^^^^^^^^^^
+This file addresses all types of users.
+
+
+OVERVIEW.rst
+============
+What is it?
+^^^^^^^^^^^
+The OVERVIEW file should contain a brief look at the key concepts and
+terms of the module (have a look at `invenio-access <https://invenio-access.readthedocs.io/en/latest/overview.html>`_)
+which should be introduced and explained in plain English.
+Advanced technical information or code examples should be avoided,
+as they will be covered in the Usage section.
+
+Who should I address?
+^^^^^^^^^^^^^^^^^^^^^
+This file addresses all types of users.
+
+
+Configuration
+=============
+
+Location: :code:`__init__.py`
```

Comment:
@lnielsen What do you think about this solution? I voted for it because in my opinion without it, it is not obvious where the configuration is located.